check.py

```diff
@@ -242,9 +242,9 @@ def check_expected(actual, expected):
                 for module, asserts in support.split_wast(wast):
                     print('        testing split module', split_num)
                     split_num += 1
-                    support.write_wast('split.wast', module, asserts)
+                    support.write_wast('split.wast', module)
                     run_opt_test('split.wast')    # also that our optimizer doesn't break on it
-                    result_wast_file = shared.binary_format_check('split.wast', verify_final_result=False, original_wast=wast)
+                    result_wast_file = shared.binary_format_check('split.wast', verify_final_result=False)
```

Comment:
These changes look correct and NFC but perhaps they could land in a separate PR?